cards.js

```diff
@@ -16,8 +28,47 @@ function shuffle(array){
   return array;
 }
 
-function buildDeck(){
-
-// Put some code in here!
+var deck = []
+function buildDeck(suits, values){
+  for(var index_v in values){
+    for(var index_s in suits){
+      var card = {suit: suits[index_s], value: values[index_v]};
+      deck.push(card);
+    }
+  }
+  return deck
+}
 
+// split deck between two players
+var player1 = []
+var player2 = []
+function dealCards(deck){
+  for(var i=0; i < deck.length; i++){
+    if (i % 2 === 0 ){
+      player1.push(deck[i]);
+    }
+    else{
+      player2.push(deck[i]);
+    }
+  }
+  return deck
 }
+
+//confirm cards are dealt cards out to players
+deck = buildDeck(suits, values)
+shuffledDeck = shuffle(deck)
+dealCards(shuffledDeck)
```

Comment:
this is fine, and depending on how explicit you want to be, you could consolidate this into
`shuffledDeck = shuffle(buildDeck(suits, values))` or even less explicit
`dealCards(shuffle(buildDeck(suits, values)))`
You can go from 3 lines of code to 1 line, but this is at the expense of code clarity and being explicit. Just wanted to provide some food for thought.